ci/push_docker.sh

```diff
@@ -24,12 +33,14 @@ function release() {
   weaviate_version="$(jq -r '.info.version' < openapi-specs/schema.json)"
   if [ "$GITHUB_REF_NAME" == "main" ]; then
     tag_exact="${DOCKER_REPO}:${weaviate_version}-${git_hash}"
+    git_branch="$GITHUB_REF_NAME"
   elif [  "$GITHUB_REF_TYPE" == "tag" ]; then
         if [ "$GITHUB_REF_NAME" != "v$weaviate_version" ]; then
             echo "The release tag ($GITHUB_REF_NAME) and Weaviate version (v$weaviate_version) are not equal! Can't release."
             return 1
         fi
         tag_exact="${DOCKER_REPO}:${weaviate_version}"
+	git_branch="$GITHUB_REF_NAME"
   else
     pr_title="$(echo -n "$PR_TITLE" | tr '[:upper:]' '[:lower:]' | tr -c -s '[:alnum:]' '-' | sed 's/-$//g')"
     if [ "$pr_title" == "" ]; then
```

Comment:
**suggestion:** I think that here when the pr_title is empty it means that we trigerred a `branch` build so here inside it should be: `git_branch="$GITHUB_REF_NAME"`